composer.json

```diff
@@ -46,7 +45,8 @@
         "vimeo/psalm": "^4.7"
     },
     "conflict": {
-        "guzzlehttp/ringphp": "<1.1.1"
+        "guzzlehttp/ringphp": "<1.1.1",
+        "zendframework/zenddiagnostics": "^1.6.0"
```

Comment:
The conflict should be with all versions

Reply:
fixed